code/__DEFINES/economy.dm

```diff
@@ -1,10 +1,10 @@
 #define STARTING_PAYCHECKS 5
 
-#define PAYCHECK_ASSISTANT 10
-#define PAYCHECK_MINIMAL 10
-#define PAYCHECK_EASY 15
-#define PAYCHECK_MEDIUM 40
-#define PAYCHECK_HARD 70
+#define PAYCHECK_ASSISTANT 30
```

Comment:
```suggestion
#define PAYCHECK_ASSISTANT 15
```
NEET

Reply:
Assistant and easy paycheck was the same one. Assistants shouldn't take 7 paychecks to get 105 credits

Reply:
if they want money then they should get a better job

Reply:
I kinda vibe with this response ngl, can we keep it at 20 at least?

Reply:
I guess, but they shouldn't be able to afford both a toolbelt and insuls roundstart